_posts/2020-04-28-changes-to-cloud-gov’s-existing-cdn-and-custom-domain-services-1.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: Changes to cloud.gov’s existing CDN and custom domain services
+date: April 28, 2020
+---
+This message is to let you know about upcoming changes to cloud.gov’s CDN and domain services.
+
+**What is happening to the old services?**
+
+Let’s Encrypt, which the cloud.gov custom domain service uses to provision SSL certificates for custom domains, is deprecating their v1 API. If you already have a domain registered, you will not be impacted, but you should still prepare for changes.
+
+**Here are the details:**
+
+cloud.gov’s existing CDN service (provided by the cdn-broker) and the custom-domain service (provided by the domains-broker) will no longer be able to provision *new domains* starting June 1, 2020. All *existing domains* will continue to operate normally.
```

Comment:
Can we use the service name defined in `cf marketplace` or a friendly name consistently? My preference would be towards the former (service name) for describing "CDN service" similar to custom-domain.